package-lock.json

```diff
@@ -12664,7 +12762,6 @@
       "requires": {
         "lodash.flattendeep": "^4.4.0",
         "nearley": "^2.7.10"
->>>>>>> 719b733... Add `enzyme` and `expect` package to help do unit testing
```

Comment:
WOW, ignore my previous comment about conflict in lockfile. Maybe they're removed by npm magically 🤔?!

Reply:
I guess it is because I removed the lockfile and reinstalled it one time